Assets/Mirror/Examples/AdditiveScenes/Scripts/AdditiveNetworkManager.cs

```diff
@@ -26,12 +28,12 @@ public void Started()
 
         IEnumerator LoadSubScenes()
         {
-            if (LogFilter.Debug) Debug.Log("Loading Scenes");
+            if (logger.LogEnabled()) logger.Log("Loading Scenes");
 
             foreach (string sceneName in subScenes)
             {
                 yield return SceneManager.LoadSceneAsync(sceneName, LoadSceneMode.Additive);
-                if (LogFilter.Debug) Debug.Log($"Loaded {sceneName}");
+                if (logger.LogEnabled()) logger.Log($"Loaded {sceneName}");
```

Comment:
In this case the if is needed because `$"Loaded {sceneName}"` is really expensive, so it is worth avoiding